README.rst

```diff
@@ -3,7 +3,6 @@ Django-SES
 ==========
 :Info: A Django email backend for Amazon's Simple Email Service
 :Author: Harry Marr (http://github.com/hmarr, http://twitter.com/harrymarr)
-:Collaborators: Paul Craciunoiu (http://github.com/pcraciunoiu, http://twitter.com/embrangler)
```

Comment:
I'd like to keep myself in the README if that's OK. Could move to Author(s) if you think that makes more sense.

Reply:
You da boss. :)